modules/configuring-distributed-workloads.adoc

```diff
@@ -10,22 +10,36 @@ To configure the distributed workloads feature for your data scientists to use i
 * You have logged in to {openshift-platform} with the `cluster-admin` role.
 * You have sufficient resources. In addition to the base {productname-short} resources, you need 1.1 vCPU and 1.6 GB memory to deploy the distributed workloads infrastructure.
 * You have access to a Ray cluster image. For information about how to create a Ray cluster, see the link:https://docs.ray.io/en/latest/cluster/getting-started.html[Ray Clusters documentation].
-* You have removed any previously installed instances of the CodeFlare Operator, as described in the Knowledgebase solution link:https://access.redhat.com/solutions/7043796[How to migrate from CodeFlare Operator to Red Hat OpenShift AI Cluster].
 ifndef::upstream[]
-* If you want to use graphics processing units (GPUs), you have enabled GPU support in {productname-short}. See link:{rhoaidocshome}{default-format-url}/managing_resources/managing-cluster-resources_cluster-mgmt#enabling-gpu-support_cluster-mgmt[Enabling GPU support in {productname-short}].
+* You have removed any previously installed instances of the CodeFlare Operator, as described in the Knowledgebase solution link:https://access.redhat.com/solutions/7043796[How to migrate from CodeFlare Operator to Red Hat OpenShift AI Cluster].
+* If you want to use graphics processing units (GPUs), you have enabled GPU support in {productname-short}.
+See link:{rhoaidocshome}{default-format-url}/managing_resources/managing-cluster-resources_cluster-mgmt#enabling-gpu-support_cluster-mgmt[Enabling GPU support in {productname-short}].
+endif::[]
+ifdef::upstream[]
+* You have removed any previously installed instances of the CodeFlare Operator.
+* If you want to use graphics processing units (GPUs), you have enabled GPU support.
+This process includes installing the Node Feature Discovery Operator and the NVIDIA GPU Operator.
+For more information, see https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html[NVIDIA GPU Operator on {org-name} OpenShift Container Platform^] in the NVIDIA documentation.
 endif::[]
 
 .Procedure
 . In the {openshift-platform} console, click *Operators* -> *Installed Operators*.
+ifndef::upstream[]
 . Search for the *Red Hat OpenShift Data Science* Operator, and then click the Operator name to open the Operator details page.
+endif::[]
+ifdef::upstream[]
+. Search for the *Open Data Hub Operator*, and then click the Operator name to open the Operator details page.
```

Comment:
Operator details should be bold as its a specific page?

*Operator details* page.

Reply:
The text "Operator details" is shown in a breadcrumb but not in the page title, so I thought it was better to use plain text (not bold) as a general description. Do you disagree?

Reply:
No. Thanks for clarifying. It's fine.